sklearn/cluster/_mean_shift.py

```diff
@@ -337,19 +337,9 @@ class MeanShift(ClusterMixin, BaseEstimator):
 
         .. versionadded:: 1.0
 
-    Examples
+    See Also
     --------
-    >>> from sklearn.cluster import MeanShift
-    >>> import numpy as np
-    >>> X = np.array([[1, 1], [2, 1], [1, 0],
-    ...               [4, 7], [3, 5], [3, 6]])
-    >>> clustering = MeanShift(bandwidth=2).fit(X)
-    >>> clustering.labels_
-    array([1, 1, 1, 0, 0, 0])
-    >>> clustering.predict([[0, 0], [5, 5]])
-    array([1, 0])
-    >>> clustering
-    MeanShift(bandwidth=2)
+    NearestNeighbors : Unsupervised learner for implementing neighbor searches.
```

Comment:
It would make more sense to associated clustering methods here like KMeans since MeanShift is a clustering method.

Reply:
Thanks for the suggestion, I have mentioned KMeans now